compiler/src/iree/compiler/InputConversion/StableHLO/Preprocessing/Canonicalization.cpp

```diff
@@ -1071,6 +1071,14 @@ struct ReorderElementwiseAndShapeOp final
           op, "defining operation of unexpected type.");
     }
 
+    // Only reorder if the defining op has no other uses.
+    auto intermediateResult = definingOp->getResult(0);
+    if (std::distance(intermediateResult.getUses().begin(),
+                      intermediateResult.getUses().end()) != 1) {
+      return rewriter.notifyMatchFailure(op,
+                                         "operation has more than one use.");
```

Comment:
no trailing . needed.

Reply:
Fixed (along with others in the pattern)